sig-contributor-experience/community-maintenance.md

```diff
@@ -0,0 +1,75 @@
+# Community Maintenance 
+
+This document outlines maintenance tasks that SIG Contributor Experience should be doing on a regular basis.
+Copy this list into a new Github issue, for example "Community Maintenance Tasks for 2Q 2018". 
+Then people can volunteer to audit different parts of the project at once. 
+These tasks should be done at a _minimum of twice a year_, but ideally once a quarter.
+
+If any of these tasks can be automated, then do so, however most of these require a human to make a judgement decision. 
+If you find that any one person is in charge of a resource it is your responsiblity to bring this issue to SIG Contributor Experience so that important parts of Kubernetes infrastructure are not assigned to one sole person. 
```

Comment:
sig-contributor-experience/community-maintenance.md:9:70: "responsiblity" is a misspelling of "responsibility"